/**/dependent_frame_publisher:
  ros__parameters:  
    gate_frame: #numbers purely for testing at the moment and does not reflect actual frame location
      parent: gman_frame
      x: 2 relative
      y: 1 relative
      z: 1 absolute
      roll: 1.5707 relative
      pitch: 0 absolute
      yaw: 3.1415 relative
    octogon_frame:
      parent: cash_frame
      x: 0 relative
      y: 0 relative
      z: 0 absolute
      roll: 0 absolute
      pitch: 0 absolute
      yaw: 0 absolute
